magento/magento2
# 33378 GraphQL: create an entry in 'customer_log' after generate customer token

### Description (*)

Create an entry in 'customer_log' after generate customer token.

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#33378

### Manual testing scenarios (*)

1. Check 'customer_log'  db table
2. Genarate customer token
```
mutation {
	generateCustomerToken(
        email: "roni_cost@example.com"
        password: "roni_cost3@example.com"
    ) {
        token
    }
}
```
3. Check 'customer_log' db table again. The field 'last_login_at' for this customer was updated.

### Questions or comments


### Contribution checklist (*)
